url.go

```diff
@@ -37,21 +37,37 @@ type URL struct {
 	Name     string // "wordpress"
 	Revision int    // -1 if unset, N otherwise
 	Series   string // "precise" or "" if unset
```

Comment:
Might be worth clarifying this actually:

    // "precise" or "" if unset; "bundle" if it's a bundle.